chat-widget/src/common/telemetry/defaultConfigs/defaultTelemetryInternalData.ts

```diff
@@ -1,7 +1,7 @@
 import { IInternalTelemetryData } from "../interfaces/IInternalTelemetryData";
 
 export const defaultInternalTelemetryData: IInternalTelemetryData = {
-    environmentVersion: "test",
+    environmentVersion: "prod",
```

Comment:
Will this start pushing all the telemetry to prod instances irrespective of environment, like ocdailym telemetry flowing to prod

Reply:
Telemetry is already pushed to prod by default. `environmentVersion` is a hardcoded value. The logger has a logic to ignore `collectorURI` if `environmentVersion` is not set to `prod`